constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/MotionLayout.java

```diff
@@ -2323,7 +2323,7 @@ private void setupConstraintWidget(ConstraintWidgetContainer base, ConstraintSet
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
                     layoutParams.resolveLayoutDirection(getLayoutDirection());
                 } else {
-                    layoutParams.resolveLayoutDirection(View.LAYOUT_DIRECTION_LTR);
+                    layoutParams.resolveLayoutDirection(0);
```

Comment:
`ConstraintLayout` seems to depend on AndroidX Core, so you could use `ViewCompat.LAYOUT_DIRECTION_LTR`